eosdatadistribution/producers/usb.py

```diff
@@ -56,7 +56,7 @@ def mount_removed_cb(monitor, mount, store):
     pprint.pprint([store.remove_name(n) for p, n in store.producers.items() if p.startswith(p)])
 
 
-if __name__ == '__main__':
+def main():
```

Comment:
I thought it was needed for the `entry_points` stuff in `setup.py`, but I admit I cargo-culted it from another module and haven’t tested it without here.